crates/rerun/Cargo.toml

```diff
@@ -51,8 +54,19 @@ log = ["dep:env_logger", "dep:log"]
 ## This adds a lot of extra dependencies, so only enable this feature if you need it!
 native_viewer = ["dep:re_viewer"]
 
+## Add support for running the rerun-cli binary via a function call.
```

Comment:
worth being concrete - what function call, what does this exactly enable for the consumer of this crate